build.gradle

```diff
@@ -37,6 +37,7 @@ dependencies {
     kapt 'org.hibernate:hibernate-jpamodelgen:5.6.7.Final'
     kapt "org.springframework.boot:spring-boot-configuration-processor"
 
+	implementation 'org.yaml:snakeyaml:1.31'
```

Comment:
Why the tab, also what does this do, do we not do dependency pinning?

Reply:
IDE was dumb, and there are several packages using snakeyaml, this updates them all to the patched version

Reply:
Is there a way to get a lock file to pin all sub dependencies?

Reply:
This is good enough for merge but iwant to have a discussion about this on discord